tests/test_xopen.py

```diff
@@ -145,6 +145,24 @@ def test_xopen_binary(fname):
         assert lines[1] == b'The second line.\n', fname
 
 
+def test_xopen_binary_no_isal_no_threads(fname, monkeypatch):
+    import xopen  # xopen local overrides xopen global variable
+    monkeypatch.setattr(xopen, "igzip", None)
```

Comment:
Can you create a fixture (`xopen_without_igzip` or so) that does the `import xopen` and `monkeypatch.setattr(...)`? That would eliminate some of the code duplication.

Reply:
Good suggestion. It is done.